index.html

```diff
@@ -6470,39 +6470,25 @@ <h3>Executing Script</h3>
 <dl class=switch>
  <dt><a>undefined</a>
  <dt><a><code>null</code></a>
- <dd><p><a>Success</a> with data <a><code>null</code></a>.
+ <dd><p><p>Return <a>success</a> with data <a><code>null</code></a>.
 
  <dt>type <a>Boolean</a>
  <dt>type <a>Number</a>
  <dt>type <a>String</a>
- <dd><p><a>Success</a> with data <var>value</var>.
-
- <dt>a <a>collection</a>
- <dd>
-  <ol>
-   <li><p>Let <var>result list</var> be an empty list.
-
-   <li><p>For each <var>item</var> of <var>value</var>,
-    add the result of <a>trying</a> to <a>clone an object</a>
-    with arguments <var>item</var> and <var>seen</var>
-    and the <a>internal JSON clone algorithm</a> as the <var>clone algorithm</var>
-    to <var>result list</var>.
-
-   <li><p>Return <a>success</a> with <var>result list</var>.
-  </ol>
+ <dd><p>Return <a>success</a> with data <var>value</var>.
 
  <dt>instance of {{Element}}</a>
  <dd><p>If the <var>element</var> <a>is stale</a>,
   return <a>error</a> with <a>error code</a> <a>stale element reference</a>.
 
-  <p>Otherwise return <a>success</a> with data set to the <a>web
+  <p>Otherwise return <a>success</a> with data <a>web
```

Comment:
So I think the reasons these ones used "set to the" is that the construct "with data [getter steps]" is more awkward than "with data [variable]". So I'd fix them more like

```
<p>Otherwise:
  <ol>
    <li><p>Let <var>reference</var> be the <a>web element reference object</var> for <var>value</var>.
    <li><p>Return <a>success</a> with data <var>reference</var>
  </ol>
```

But this is of course way more verbose.

Reply:
I'm fine with that change, and will apply it.